README.md

```diff
@@ -0,0 +1,74 @@
+<h1 align="center" style="margin:1em;">
+  <a href="https://iris-grib.readthedocs.io/en/latest/">iris-grib</a>
+</h1>
+
+<h4 align="center">
```

Comment:
Having non-contiguous heading levels (that is, an h3 tag should never directly follow an h1 tag etc., but can follow another h3, or lower level tag) is bad form as it reduces web page accessibility for tools such as screen readers.